<launch>

  <!--  Arm controllers for mannequin mode -->
  <rosparam command="load" file="$(find pr2_mannequin_mode)/config/pr2_arm_controllers_loose.yaml" />
  <node pkg="pr2_controller_manager" type="spawner" name="spawn_arm_controllers"
        args="--stopped r_arm_controller_loose l_arm_controller_loose" />

  <!-- Head controllers for mannequin mode -->
  <rosparam command="load" file="$(find pr2_mannequin_mode)/config/head_position_controller_loose.yaml" />
  <node pkg="pr2_controller_manager" type="spawner" name="spawn_head_controller"
        args="--stopped head_traj_controller_loose" />
  
  <!-- Trajectory Locks for Mannequin Modes -->
  <include file="$(find pr2_mannequin_mode)/launch/trajectory_lock.launch"/>

  <node pkg="pr2_tuck_arms_action" type="tuck_arms.py" name="tuck_arms_action" output="screen">
  	<param name="controller_name" type="string" value="arm_controller"/>
  </node>

  <node pkg="pr2_teleop_general" type="pr2_teleop_general_keyboard" name="pr2_teleop_general_keyboard" output="screen">
  	<param name="control_body" type="bool" value="true"/>
  	<param name="control_head" type="bool" value="true"/>
  	<param name="control_larm" type="bool" value="true"/>
  	<param name="control_rarm" type="bool" value="true"/>
  	<param name="control_prosilica" type="bool" value="false"/>
	<param name="tilt_diff" type="double" value=".05" />
	<param name="pan_diff" type="double" value=".05" />
	<param name="torso_step" type="double" value=".01" />
	<param name="arm_x_scale" type="double" value=".10" />
	<param name="arm_y_scale" type="double" value=".10" />
	<param name="arm_z_scale" type="double" value=".10" />
  </node>

</launch>
